database_adapters/db_adapters.py

```diff
@@ -137,6 +137,18 @@ def get_paginated_collection_data(
         )
         return list(result)
 
+    def get_collection_all_data(self, database_name, collection_name):
```

Comment:
In line 97 of this file, there is the method `get_all_collection_data`. Let's use that and delete this `get_collection_all_data`, as it is duplicated code.